store/refresh_store.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+workdir=$(realpath "$(dirname "$0")")
+log=$workdir/refresh_store.log
+
+cd "$workdir"
+date >> "$log"
+git pull &>/dev/null
+cat cron | sed "s@__BASEDIR__@$workdir@g" > /etc/cron.d/refresh_store
```

Comment:
Uuuuuh :thinking: 

Not sure to understand, is this script supposed to be ran from the cron job :thinking:  ? (https://github.com/YunoHost/apps/blob/master/cron)